addon/components/basic-dropdown/content.js

```diff
@@ -117,5 +142,17 @@ export default WormholeComponent.extend({
       dropdown.removeEventListener('DOMNodeInserted', reposition);
       dropdown.removeEventListener('DOMNodeRemoved', reposition);
     }
+  },
+
+  touchMoveHandler() {
+    const appRoot = this.get('appRoot');
+    if (!this.isDestroying && !this.isDestroyed) {
+      this.hasMoved = true;
+      appRoot.removeEventListener('touchmove', this.touchMoveHandler.bind(this));
```

Comment:
This is not detaching the event properly. Every time you use `Function.prototype.bind` your are generating a new function, so this event handler you are removing is not the same event handler you attached a few lines below.

You need to, on init, bind it and store it in a propertly, like you already did with `touchStartHandler`